### ISSUE TITLE
 Failed to make response of 'getLastHealingData' request.  java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:7878[Need support]: 

### ISSUE THREAD

**Issue by priyanka748** [healenium/healenium-web#170]
### Describe the problem

Session ID: 6f6bf4b39c2c6a256baa86fa858de247
Trying to heal...
[main] WARN com.epam.healenium.client.RestClient - Failed to make response of 'getLastHealingData' request. 
java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:7878

- Error during findElement: 
java.lang.NullPointerException
Failed to find element using By.name:

### Healenium Web version

	<version>3.2.1</version>

### Healenium Backend version

hlm-backend:3.1.5

### Selenium version

4.1.1

### Platform

Java

### Properties file

recovery-tries = 3
heal-enabled = true
serverHost = 5432
serverPort = 7878
imitatePort = 8000
score-cap = 0.5

### Logs appeared during using Healenium

```shell
Trying to heal...
[main] WARN com.epam.healenium.client.RestClient - Failed to make response of 'getLastHealingData' request. 
java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:7878

Exception in thread "main" org.openqa.selenium.NoSuchElementException: Failed to find element using By.name: q
```

**Comment by ElenaStepuro** [comment]
Hello @priyanka748 , `healenium-web 3.2.1` not compatible with back 3.1.5. Please, use `healenium-back version 3.2.0`

**Comment by priyanka748** [comment]
<img width="379" alt="Capture" src="https://user-images.githubusercontent.com/49894816/151812174-1bd3c45b-1404-4603-b0e3-eacaea4d9fb9.PNG">

serverHost = 5432
serverPort = 7878
imitatePort = 8000

is that correct??

**Comment by ElenaStepuro** [comment]
@priyanka748 `serverPort = 7878` and `imitatePort = 8000` are correct!
Please, change serverHost to yours. If you're running tests locally then `serverHost = localhost`

**Comment by priyanka748** [comment]
WARN com.epam.healenium.client.RestClient - Failed to make response of 'selectorsRequest' request.java.io.IOException: unexpected end of stream on http://localhost:7878/...
still getting this

**Comment by priyanka748** [comment]
<img width="608" alt="cap" src="https://user-images.githubusercontent.com/49894816/151816566-c37fdb5c-a1e0-4433-8c13-fe0bfaea2aa6.PNG">
is this correct?

**Comment by ElenaStepuro** [comment]
@priyanka748 please, change healenium/db/sql folder structure. So, it should be like this:
![image](https://user-images.githubusercontent.com/20905103/151818311-cc1a80c4-27e8-4301-905c-43b4024e530a.png)


So, you can just copy-paste from example: https://github.com/healenium/healenium-example-maven
And instead of `infra `you'll have `healenium/db/sql`

**Comment by priyanka748** [comment]
[main] WARN com.epam.healenium.processor.FindElementProcessor - Failed to find an element using locator By.name: q

still same

**Comment by ElenaStepuro** [comment]
@priyanka748 this is Healenium logs) like a notification that your locator hasn't been found and Healenium trying to find a new one)
`[main] WARN com.epam.healenium.processor.FindElementProcessor - Failed to find an element using locator By.name: q`

**Comment by priyanka748** [comment]
![image](https://user-images.githubusercontent.com/49894816/151839088-e0f8ad59-e8a5-45dd-909c-fc3bac98ac7b.png)
In report this is showing.. m not able to see screenshot

**Comment by priyanka748** [comment]
Working as expected

**Comment by bjha1** [comment]
Hi Team, I am getting an error -
"Trying to heal...
[main] WARN com.epam.healenium.client.RestClient - Failed to make response of 'getLastHealingData' request. 
java.io.IOException: unexpected end of stream on http://localhost:7878/..."
<img width="969" alt="Screenshot 2023-01-14 at 5 15 01 AM" src="https://user-images.githubusercontent.com/118807278/212467045-0a372c1c-a70d-4a1e-a06f-46c7311f84b6.png">

**Comment by bjha1** [comment]
> Hi Team, I am getting an error - "Trying to heal... [main] WARN com.epam.healenium.client.RestClient - Failed to make response of 'getLastHealingData' request. java.io.IOException: unexpected end of stream on http://localhost:7878/..." <img alt="Screenshot 2023-01-14 at 5 15 01 AM" width="969" src="https://user-images.githubusercontent.com/118807278/212467045-0a372c1c-a70d-4a1e-a06f-46c7311f84b6.png">
<img width="307" alt="Screenshot 2023-01-15 at 4 42 33 AM"
<img width="947" alt="Screenshot 2023-01-15 at 4 42 51 AM" src="https://user-images.githubusercontent.com/118807278/212533813-9c031bfb-567f-44db-b915-89894b749de0.png">
 src="https://user-images.githubusercontent.com/118807278/212533800-19433ff0-7bdf-4de4-a917-dc95c4ac6d3a.png">

Please help on this I have followed every steps of hlm backend even though its not working

**Comment by AkashSai10** [comment]
hi,

I am unable to connect to port 5432.
Getting below error: 

Starting selector-imitator ... 
Recreating 6216c076489f_postgres-db ... error
Starting selector-imitator          ... done
ERROR: for 6216c076489f_postgres-db  Cannot start service postgres-db: Ports are not available: exposing port TCP 0.0.0.0:5432 -> 0.0.0.0:0: listen tcp 0.0.0.0:5432: bind: address already in use

ERROR: for postgres-db  Cannot start service postgres-db: Ports are not available: exposing port TCP 0.0.0.0:5432 -> 0.0.0.0:0: listen tcp 0.0.0.0:5432: bind: address already in use
ERROR: Encountered errors while bringing up the project.

Shall i use any other port, will that work?
